rules/mech-building/validating.md

```diff
@@ -5,54 +5,49 @@ permalink: rules/build-validating/
+### Component Placement
+Double check that:
+- You only have one cabin
+- Only attachments (weapons and equipment) are in the arm and leg sections
+- Large components are in large slots 
 
-### Resource Calculations
-Let's start by breaking down resources into three categories: **Available Resources**, **Used Resources**, and **Excess Resources**. These should add up such that **Available = Used + Excess**. 
-
-#### Available Resources
-Available resources are the total amount of each resource you have to work with. To calculate Available resources, you just need to figure out how much of each resource your mech is producing. There are some exceptions, but usually that breaks down something like this: 
+### Power Type
+Before calculating any resources it's a good idea to get the hard constraints out of the way. First up is {% keyword "Power Type" glossary %}. If you are doing a single power plant mech (or a multi-power plant mech but they all produce the same power type), then all your other components must match the power type of your power plant. For mechs with multiple power types see [Multiple Power Plants]({{site_url}}/rules/multiple-power-plants/).
 
-| Resource   | Available amount                                             |
-| ---------- | ------------------------------------------------------------ |
-| Power      | Amount produced by Power Plant                               |
-| Cooling    | Amount provided by Frame & Plating + any provided by Coolers |
-| Processing | Amount provided by Mainframe + any provided by Modules       |
+### Software
+Your cabin and all your modules must be compatible with the {% keyword "Software" glossary %} of your mainframe.
 
-{.table}
-
-Note that each section of your mech has its own plating, so you get Cooling from the plating on each section. 
+### Resource Calculations
+There are three parts to {% keyword "Resources" glossary %}, {% keyword "Available Resources" glossary %} (produced by components), {% keyword "Used Resources" glossary %} (used by constant-draw components), and {% keyword "Excess Resources" glossary %} (left over for excess draw components). They are related like so:
 
-As you can see from the table above, your resources mostly come from **Infrastructure Components** such as your mech's Frame, Plating, Power Plant, and Mainframe. 
+Excess Resources = Available Resources - Used Resources
 
-<!-- TODO: Add example -->
+To make sure you have enough resources simply go through your list of components and sum up the available and used resources and calculated the remaining excess resources. Then check to make sure your excess resources is enough to use as many of your excess-draw components as you want to in a single turn.
```

Comment:
A couple of things here:
- I think a comma after the 1st "resources" would be nice
- calculate~~d~~

Reply:
Done